kola/tests/ignition/resource.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"fmt"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+	"github.com/coreos/mantle/platform/conf"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "coreos.ignition.v2_1.resource.local",
+		Run:         resourceLocal,
+		ClusterSize: 1,
+		UserData: conf.Ignition(`{
+		  "ignition": {
+		      "version": "2.1.0"
+		  },
+		  "storage": {
+		      "files": [
+			  {
+			      "filesystem": "root",
+			      "path": "/resource/data",
+			      "contents": {
+				  "source": "data:,kola-data"
+			      }
+			  }
+		      ]
+		  }
+	      }`),
+	})
+	register.Register(&register.Test{
+		Name:             "coreos.ignition.v2_1.resource.remote",
+		Run:              resourceRemote,
+		ClusterSize:      1,
+		ExcludePlatforms: []string{"qemu"},
```

Comment:
Why exclude `qemu`?

Reply:
Jenkins runs it without Internet access.